code/game/machinery/Sleeper.dm

```diff
@@ -134,6 +134,33 @@
 		else
 			user << "<span class='warning'>\The [src] has a beaker already.</span>"
 		return
+	else if(istype(I, /obj/item/weapon/grab))
+
+		var/mob/living/L = I:affecting

```

Comment:
This var is only used once, whereas it could be used to replace all `I:affecting` checks after this point.

Also, since this operation doesn't require speed, you may want to typecast I. Usage of the unsafe : access method is not recommended for standard pieces of code, as it bypasses the compiler safeties. Only applicable arguments are speed, like in ismob and whatever definitions.

So, before this line,
```
var/obj/item/weapon/grab/G = I
```

And then replace all `I:` that you're left with with `G.` That way, the compiler will yell at us if that variable doesn't exist.